cpp/src/lance/format/schema.h

```diff
@@ -190,6 +193,8 @@ class Field final {
 
   bool RemoveChild(int32_t id);
 
+  int32_t GetFieldsCount() const;
```

Comment:
can you add a quick code comment to clarify whether this include the field itself

Reply:
Sure will do